_includes/head.html

```diff
@@ -10,7 +10,7 @@
   <meta name="robots" content="{{ page.robots }}">
   {%- endif -%}
   {%- seo -%}
-  <link rel="icon" type="image/svg+xml" href="/static/img/libera-color.svg">
+  <link rel="icon" type="image/svg+xml" alt="Libera Logo" href="/static/img/libera-color.svg">
```

Comment:
I don't think link elements should have an alt property? That's not a thing this element supports https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link

Reply:
Whoops, yes my mistake, not required at all!